src/overlays/actors/ovl_Obj_Tokei_Tobira/z_obj_tokei_tobira.c

```diff
@@ -38,14 +39,142 @@ static InitChainEntry D_80ABD750[] = {
 
 #endif
 
-extern InitChainEntry D_80ABD750[];
+extern InitChainEntry D_80ABD750;
+extern Vec3f D_80ABD760;
+extern s16 D_80ABD76C[];
+extern s32 D_80ABD770;
```

Comment:
How

Reply:
You need to fix  some of the types, like the variables that should be arrays but are currently not arrays.
m2c can help you migrating the data, you need to update your ctx file and then pass both your ctx and the `.data.s` file to  m2c.
Otherwise you can manually copy the asm data into the C file and fix it appropriately